src/guacscanner/guacscanner.py

```diff
@@ -0,0 +1,548 @@
+  --postgres-password-file=FILENAME    The file from which the PostgreSQL password will be read. [default: /run/secrets/postgres-password]
+  --postgres-username=USERNAME    If specified then the specified value will be used when connecting to the PostgreSQL database.  Otherwise, the username will be read from a local file.
+  --postgres-username-file=FILENAME    The file from which the PostgreSQL username will be read. [default: /run/secrets/postgres-username]
+  --private-ssh-key=KEY  If specified then the specified value will be used for the private ssh key.  Otherwise, the ssh key will be read from a local file.
+  --private-ssh-key-file=FILENAME  The file from which the private ssh key will be read. [default: /run/secrets/private-ssh-key]
+  --rdp-password=PASSWORD  If specified then the specified value will be used for the RDP password.  Otherwise, the password will be read from a local file.
+  --rdp-password-file=FILENAME  The file from which the RDP password will be read. [default: /run/secrets/rdp-password]
+  --rdp-username=USERNAME  If specified then the specified value will be used for the RDP username.  Otherwise, the username will be read from a local file.
+  --rdp-username-file=FILENAME  The file from which the RDP username will be read. [default: /run/secrets/rdp-username]
+  --sleep=SECONDS  Sleep for the specified number of seconds between executions of the Guacamole connection update loop. [default: 60]
+  --vnc-password=PASSWORD  If specified then the specified value will be used for the VNC password.  Otherwise, the password will be read from a local file.
+  --vnc-password-file=FILENAME  The file from which the VNC password will be read. [default: /run/secrets/vnc-password]
+  --vnc-username=USERNAME  If specified then the specified value will be used for the VNC username.  Otherwise, the username will be read from a local file.
+  --vnc-username-file=FILENAME  The file from which the VNC username will be read. [default: /run/secrets/vnc-username]
+  --vpc-id=VPC_ID        If specified then query for EC2 instances created
+                         or destroyed in the specified VPC ID.  If not
+                         specified then the ID of the VPC in which the host
+                         resides will be used.
+"""
+
+
+# Standard Python Libraries
+import logging
+import re
+import sys
+import time
+
+# Third-Party Libraries
+import boto3
+import docopt
+from ec2_metadata import ec2_metadata
+import psycopg
+from psycopg import sql
+from schema import And, Optional, Or, Schema, SchemaError, Use
+
+from ._version import __version__
+
+DEFAULT_ADD_INSTANCE_STATES = [
+    "running",
+]
+DEFAULT_POSTGRES_DB_NAME = "guacamole_db"
+DEFAULT_POSTGRES_HOSTNAME = "postgres"
+DEFAULT_POSTGRES_PORT = 5432
+DEFAULT_REMOVE_INSTANCE_STATES = [
+    "terminated",
+]
+DEFAULT_AMI_SKIP_REGEXES = [
+    re.compile(r"^guacamole-.*$"),
```

Comment:
Skip Nessus AMIs as well- those instances are only accessible thru a browser on another instance (e.g. Kali or Teamserver):
```suggestion
    re.compile(r"^guacamole-.*$"),
    re.compile(r"^nessus-.*$"),
```